src/steps/access.ts

```diff
@@ -31,31 +43,46 @@ export async function fetchUsers({
   )) as Entity;
 
   await apiClient.iterateUsers(async (user) => {
+    const employeeEntity = await getEmployeeEntity(jobState, user);
+    const employee = employeeEntity
+      ? <BambooHREmployee>getRawData(employeeEntity)
+      : undefined;
+
+    if (employeeEntity && !employee) {
```

Comment:
Is there ever a case where we don't want to throw an issue on this and want to just keep ingesting?

Reply:
That's a great question. As I considered it when writing this check, I figured that something is seriously wrong if there is an entity that doesn't have the raw data. In fact, I questioned the value of adding this check at all but figured, since I've seen it written in other integrations, I should probably add the check.